operations/mimir/store-gateway.libsonnet

```diff
@@ -36,6 +36,9 @@
       // Relax pressure on KV store when running at scale.
       'store-gateway.sharding-ring.heartbeat-period': '1m',
     } +
+    (if 'store_gateway_grpc_max_query_response_size_bytes' in $._config then {
```

Comment:
Why make this conditional on the config existing when it's explicitly set in `config.libsonnet`?

Reply:
I guess I was considering it an optional config, so wanted to hedge against it possibly not being set at all. If you think it's unnecessary, I'll happily remove it (and the similar change in the helm charts).